### ISSUE TITLE
Running node-cluster preset build throwing EADDRINUSE errors

### ISSUE THREAD

**Issue by aisteneciunaitevalantic** [nuxt/nuxt#29964]
### Environment

------------------------------
- Operating System: Linux
- Node Version:     v20.10.0
- Nuxt Version:     3.14.159
- CLI Version:      3.15.0
- Nitro Version:    2.10.4
- Package Manager:  pnpm@8.14.1
- Builder:          -
- User Config:      default
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### Reproduction

https://stackblitz.com/edit/github-zvxiom?file=nuxt.config.ts

```
> npm run build
> node .output/server/index.mjs
```

### Describe the bug

Running application with preset 'node-cluster' is throwing EADDRINUSE errors

```
[nitro] [uncaughtException] Error: bind EADDRINUSE null:3000
    at listenOnPrimaryHandle (node:net:1940:18)
    at rr (node:internal/cluster/child:163:12)
    at Worker.<anonymous> (node:internal/cluster/child:113:7)
    at process.onInternalMessage (node:internal/cluster/utils:49:5)
    at process.emit (node:events:526:35)
    at emit (node:internal/child_process:951:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -98,
  code: 'EADDRINUSE',
  syscall: 'bind',
  address: null,
  port: 3000
}
```

### Logs

```shell-script

```

**Comment by danielroe** [comment]
Yes, this is reproducible for me. I think it's an upstream nitro issue - would you see if you can reproduce this in pure Nitro ([reproduction sandbox](https://stackblitz.com/github/unjs/nitro/tree/main/examples/hello-world)), and if so, raise [there](https://github.com/unjs/nitro/issues/new/choose)? 🙏

**Comment by aisteneciunaitevalantic** [comment]
I could reproduce in Nitro, here's an upstream ticket: https://github.com/nitrojs/nitro/issues/2892

**Comment by Zae** [comment]
This has been an issue for over an month now, @pi0 's fix to use the nightly version of nitro is not really a possibility in production, if you want to use cluster mode we are now stuck on `3.13`.

Is there another way to fix this issue?

**Comment by danielroe** [comment]
you can patch nitro or use a custom preset (you can copy the one from nitro with the fix)

**Comment by Zae** [comment]
I'd rather not use patches or nighties.

Do you know when you expect the cluster preset to work again in nuxt stable?

**Comment by danielroe** [comment]
I think Pooya plans a patch soon, but that’s really a question for him.
